wkcuber/wkcuber/convert_raw.py

```diff
@@ -0,0 +1,239 @@
+import argparse
+from functools import partial
+import logging
+from pathlib import Path
+import time
+from typing import Optional, Tuple, Union
+import numpy as np
+
+from webknossos.dataset.defaults import DEFAULT_WKW_FILE_LEN
+from webknossos import BoundingBox, Dataset, Mag, MagView
+from webknossos.utils import get_executor_for_args, wait_and_ensure_success
+from .utils import (
+    add_distribution_flags,
+    add_interpolation_flag,
+    add_sampling_mode_flag,
+    add_scale_flag,
+    add_verbose_flag,
+    get_executor_args,
+    setup_logging,
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+CHUNK_SIZE = 1024, 1024, 1024
```

Comment:
Is there a place I can get the size of the block files from instead of having it hard-coded here?

Reply:
I think it would make sense to the chunk size dependent on the output wkw file sizes (default: 1024^3). You can compute that from `(block_len * file_len,) * 3`. We don't expose a constant for `block_len` atm, which we probably should, but it is always 32.

Reply:
I just update this, thanks!